apps/server/src/modules/files-storage/files-storage.module.ts

```diff
@@ -0,0 +1,61 @@
+import { Module } from '@nestjs/common';
+import { S3Client } from '@aws-sdk/client-s3';
+import { MikroOrmModule } from '@mikro-orm/nestjs';
+import { ALL_ENTITIES } from '@shared/domain';
+import { Configuration } from '@hpi-schul-cloud/commons';
+import { LoggerModule } from '@src/core/logger';
+import { FileRecordRepo } from '@shared/repo';
+import { FilesStorageController } from './controller/files-storage.controller';
+import { S3ClientAdapter } from './client/s3-client.adapter';
+import { S3Config } from './interface/config';
+import { AuthModule } from '../authentication/auth.module';
```

Comment:
should not be imported relatively. should be `...from '@src/modules/authentication/auth.module'`